src/Components/Slider/Slider.js

```diff
@@ -0,0 +1,158 @@
+import React, { useState, useEffect, useRef } from 'react';
+import styled from 'styled-components';
+import { SliderContainer } from './SliderContainer';
+import { Slide } from './Slide';
+import { Arrow } from './Arrow';
+import { Dots } from './Dots';
+
+const SliderCSS = styled.div`
+  position: relative;
+  height: 100vh;
+  width: 100vw;
+  margin: 0 auto;
+  overflow: hidden;
+  white-space: nowrap;
+`
+
+const getWidth = () => window.innerWidth
+
+/**
+ * @function Slider
+ */
+const Slider = props => {
+
+  const slides  = Object.values(props);
```

Comment:
Not the best use case for the props. Please refer to my previous comment on a better way to pass down props to this component.